source/js/scripts/app.js

```diff
@@ -280,6 +280,9 @@ function handleOnLoad() {
   if (!backend.get('Username')) {
     window.location.href = 'index.html';
   } else if (backend.get('Started')) {
+    // On resuming session, reset timer state and current pomos
```

Comment:
Maybe we should reset these values in `StartButtons.js` when the user clicks the Resume My Session button so they won't get reset on a reload